src/cli.ts

```diff
@@ -127,7 +127,7 @@ async function getToolLocation(cmd): Promise<string> {
         } catch (error) {
             const response = await vscode.window.showInformationMessage(
                 `Cannot find ${tools[cmd].description}.`, 'Download and install', 'Help', 'Cancel');
-            if(response === 'Download and install') {
+            if (response === 'Download and install') {
```

Comment:
Please do not mix fix with formatting changes.

Reply:
sure. will keep this in mind :)